src/03_modules.py

```diff
@@ -10,22 +11,27 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(sys.argv[0])
```

Comment:
This wants you to print out each argument from the command line on each line. If you were to type in the command line python3 03_modules.py dog cat bird, it would print

03_modules.py
dog
cat
bird

I would suggest doing this by using

`for arg in sys.argv: print(arg)`